#request
uint8 payload_index
---
#response
string camera_name
